extension/app/scripts.babel/background.js

```diff
@@ -17,12 +17,18 @@
 
 'use strict';
 
-import {runPwaAudits} from './pwa-check.js';
+const ExtensionProtocol = require('../../../helpers/extension/driver.js');
```

Comment:
I thought we'd settled on require. AFAIK this is the only case where we had import, and that's now gone.